dcs_deploy.py

```diff
@@ -7,17 +7,53 @@
 import wget
 from threading import Thread, Event
 import time
+from urllib.parse import urlparse
+import sys as _sys
+
+dcs_deploy_version = "0.1.0"
```

Comment:
Maybe we should use main version `2` if we call it release 2?
```suggestion
dcs_deploy_version = "2.1.0"
```